src/hotspot/share/logging/logDecorators.cpp

```diff
@@ -25,6 +25,10 @@
 #include "logging/logDecorators.hpp"
 #include "runtime/os.hpp"
 
+const LogLevelType AnyLevel = LogLevelType::NotMentioned;
+#define UNDECORATED_DEFAULTS \
+  UNDECORATED_DEFAULT(AnyLevel, LOG_TAGS(jit, inlining))
```

Comment:
Maybe move this down to next where it is used and then `#undef UNDECORATED_DEFAULTS`